godtools/App/Share/Domain/UseCases/GetLanguageAvailabilityStringUseCase/GetLanguageAvailablityStringUseCase.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  GetLanguageAvailablityStringUseCase.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 7/18/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import Foundation
+
+class GetLanguageAvailabilityStringUseCase {
+    
+    let localizationServices: LocalizationServices
+    let getTranslatedLanguageUseCase: GetTranslatedLanguageUseCase
+    
+    init(localizationServices: LocalizationServices, getTranslatedLanguageUseCase: GetTranslatedLanguageUseCase) {
+        self.localizationServices = localizationServices
+        self.getTranslatedLanguageUseCase = getTranslatedLanguageUseCase
+    }
+    
+    func getLanguageAvailability(for resource: ResourceModel, language: LanguageModel?) -> (isAvailable: Bool, string: String) {
```

Comment:
Hey @rachaelblue the only change I would suggest is not providing a return tuple type here and instead provide a DomainModel to wrap those business attributes.  However, leave the tuple for now because this UseCase will be re-visited when the clean up sprint happens.  This UseCase may even go away and the logic in here will be part of a bigger UseCase for fetching a ToolDomainModel which I will explain at a later time.

Reply:
Sounds good!